pkgs/development/python-modules/pyside6/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, cmake
+, ninja
+, qt6
+, python
+, shiboken6
+, libxcrypt
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pyside6";
+
+  inherit (shiboken6) version src;
+
+  postPatch = ''
+    # dont ignore optional qt modules
+    substituteInPlace sources/pyside6/cmake/PySideHelpers.cmake \
+      --replace \
+        'string(FIND "''${_module_dir}" "''${_core_abs_dir}" found_basepath)' \
+        'set (found_basepath 0)'
+
+    cd sources/${pname}
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  buildInputs = [
+    python
```

Comment:
Should `python` be in `nativeBuildInputs` instead?

Reply:
Changed `python` to be in `nativeBuildInputs` to match `pyside2` :+1: